PlandoRandomSettings.py

```diff
@@ -127,18 +161,22 @@ def getRandomFromType(setting):
         random_settings[info.name] = getRandomFromType(info)
         print(info.name + ' : ' + info.gui_type + ' : ' + str(getRandomFromType(info)))
 
-# Randomize the starting items
+# Randomize the starting items and songs in legacy mode
 if STARTING_ITEMS == "legacy":
-    starting_items_dict = {}
+    starting_items = []
+    starting_songs = []
     fast_travel = R.choice([True, False])
     omega_wallet = R.choice([True, False])
     if fast_travel:
-        starting_items_dict.update({"Farores Wind": 1, "Prelude of Light": 1, 
-                                    "Serenade of Water": 1})
+        starting_items.append("farores_wind")
+        starting_songs.append("prelude")
+        starting_songs.append("serenade")
     if omega_wallet:
-        starting_items_dict.update({"Progressive Wallet": 3})
-    random_settings["starting_items"] = starting_items_dict
-
+        starting_items.append("wallet")
+        starting_items.append("wallet2")
+        starting_items.append("wallet3")
```

Comment:
Turns out the expected format is an array of settingnames, e.g., wallet, wallet2, wallet3. If you include just "wallet2" it is properly understood to be just one wallet upgrade and shows up as the adult wallet.